user/views.py

```diff
@@ -1,3 +1,68 @@
+import datetime
+import json
+import bcrypt
+import jwt
+
 from django.shortcuts import render
+from django.views     import View  
+from django.http      import HttpResponse, JsonResponse
+
+from my_settings      import SECRET_KEY
+from user.models      import User, Gender, SkinInfo
+
+
+class SignUpView(View):
+    def post(self, request):
+        user_data = json.loads(request.body)
+        try:
+            if User.objects.filter(account = user_data['account']).exists():
+                return JsonResponse({'message':'DUPLICATED ID'}, status=400)
+
+            User.objects.create(
+                name        = user_data['name'],
+                account     = user_data['account'],
+                password    = bcrypt.hashpw(user_data['password'].encode('utf-8'), bcrypt.gensalt()).decode(),
+                email       = user_data['email'],
+                birthday    = datetime.datetime.strptime(user_data['birthday'], "%Y%m%d").date(),
+                gender_type = Gender.objects.get(name = user_data['gender_type']),
+                phoneNumber = user_data['phoneNumber'],
+            ).save()
+
+
+            SkinInfo(
+                skinType        = user_data['skinType'],
+                skinTrouble     = user_data['skinTrouble'],
+                skinSensitivity = user_data['skinSensitivity']
+            ).save()
+
+
+            return JsonResponse({'message':'SIGNUP SUCCESS'}, status=200)
+        
+        except KeyError:
+            return JsonResponse({'message':'KEYERROR'}, status=400)
+
+class SignInView(View):
+    def post(self, request):
+        user_data = json.loads(request.body)
+
+        try:
+            if User.objects.filter(account = user_data['account']).exists():
+                user = User.objects.get(account = user_data['account'])
+
+                if bcrypt.checkpw(user_data['password'].encode('utf-8'), user.password.encode('utf-8')):
+                    token = jwt.encode({'user_id':user.id}, 'secret', algorithm='HS256')
+                    token = token.decode('utf-8')
+                    return JsonResponse({'access_token':token}, status=200)
+                else:
+                    return JsonResponse({'message':'UNAUTHORIZED'}, status=401)
```

Comment:
```suggestion
                if bcrypt.checkpw(user_data['password'].encode('utf-8'), user.password.encode('utf-8')):
                    token = jwt.encode({'user_id':user.id}, 'secret', algorithm='HS256')
                    token = token.decode('utf-8')
                    return JsonResponse({'access_token':token}, status=200)
            
                return JsonResponse({'message':'UNAUTHORIZED'}, status=401)
```
`if`문을 만족하지 못해 return 200에 도달하지 못한다면 바로 다음 줄인 아래 Unauthorized 코드에 도달하는 로직이죠? `else`문이 없어도 `if` 조건을 만족하지 못한다면 바로 아랫 줄 401 return으로 올 수 있으므로, 보다 나은 코드를 위해 꼭 있어야 하는 `else`가 아니라면 제거하시는 것을 추천드립니다 :)